autogluon/task/tabular_prediction/hyperparameter_configs.py

```diff
@@ -0,0 +1,49 @@
+
+import copy
+
+
+# Dictionary of preset hyperparameter configurations.
+hyperparameter_config_dict = dict(
+    # Default AutoGluon hyperparameters intended to maximize accuracy without significant regard to inference time or disk usage.
+    default={
+        'NN': {},
+        'GBM': {},
+        'CAT': {},
+        'RF': {},
+        'XT': {},
+        'KNN': {},
+        'custom': ['GBM'],
+    },
+    # Results in smaller models. Generally will make inference speed much faster and disk usage much lower, but with worse accuracy.
+    light={
```

Comment:
Do we want to constrain max_size of weightedensemble in light?

Reply:
It is defaulted to 5 max from a given model type and 25 max overall. These hyperparameters are not surfaced to users at present, but we can add them at a later time.